### PR TITLE
Augmented combinational loop report

### PR DESCRIPTION
**Pull request by chick**
When found during symbol table processing It displays a combinational loop
(which will just be the first one found, if there are more than one).
The path contains the statements and the @INFO path with symbols on path in red
If a combinational loop is found by CheckComboLoops and you want to see this enhanced display, turn off the combination loop checking with an annotation or command line flag.
DontCheckCombLoopsAnnotation or --no-check-comb-loops

### CODE REVIEW

File: src/test/scala/treadle/executable/SymbolTableSpec.scala [hunk 1]
```diff
@@ -16,15 +16,18 @@ limitations under the License.
...
 import firrtl.stage.FirrtlSourceAnnotation
 import firrtl.transforms.DontCheckCombLoopsAnnotation
+import logger.{LazyLogging, LogLevel, Logger}
 import treadle._
 import org.scalatest.freespec.AnyFreeSpec
 import org.scalatest.matchers.should.Matchers
 
 //scalastyle:off magic.number
-class SymbolTableSpec extends AnyFreeSpec with Matchers {
+class SymbolTableSpec extends AnyFreeSpec with Matchers with LazyLogging {
```
Comment (ucbjrl) [inline src/test/scala/treadle/executable/SymbolTableSpec.scala:30]: Should the `LazyLogging` changes be a separate PR?

Comment (chick) [reply src/test/scala/treadle/executable/SymbolTableSpec.scala]: Done
